src/store/thunks/movieDetails.js

```diff
@@ -0,0 +1,15 @@
+import axios from "../../core/axiosHelper";
```

Comment:
This should be an instance of Axios, otherwise you're doing mutation which is not easily supported in larger apps.

Reply:
instance added and imported